fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/IndexExpansionInfo.java

```diff
@@ -51,18 +52,22 @@ public final class IndexExpansionInfo {
     private final Collection<RecordType> indexedRecordTypes;
     @Nonnull
     private final Set<String> indexedRecordTypeNames;
+    @Nonnull
+    private final Type.Record plannerBaseType;
```

Comment:
`baseType` sufficient? I mean is there a difference of `baseType` in all the candidate implementations and this `baseType`?